src/tests/java/htsjdk/samtools/util/SamLocusIteratorTest.java

```diff
@@ -161,10 +196,91 @@ public void testSimpleGappedAlignment() {
         };
         int i = 0;
         for (final SamLocusIterator.LocusInfo li : sli) {
-            Assert.assertEquals(li.getRecordAndPositions().size(), 2);
+            Assert.assertEquals(li.getRecordAndPositions().size(), coverage);
             Assert.assertEquals(li.getPosition(), expectedReferencePositions[i]);
             Assert.assertEquals(li.getRecordAndPositions().get(0).getOffset(), expectedReadOffsets[i]);
             Assert.assertEquals(li.getRecordAndPositions().get(1).getOffset(), expectedReadOffsets[i]);
+            // make sure that we are not accumulating indels
+            Assert.assertEquals(li.getDeletedInRecord().size(), 0);
+            Assert.assertEquals(li.getInsertedInRecord().size(), 0);
+            ++i;
+        }
+    }
+
+    /**
+     * Try all CIGAR operands (except H and P) and confirm that loci produced by SamLocusIterator are as expected with indels
+     */
+    @Test
+    public void testSimpleGappedAlignmentWithIndels() {
```

Comment:
would be better to include this in a loop over `includeIndels` as you did in other tests.